src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let amount = days * 40;
+  const firstDiscount = 20;
+  const secondDiscount = 50;
+
+  if (days >= 7) {
+    amount -= secondDiscount;
+  } else if (days >= 3) {
+    amount -= firstDiscount;
+  }
```

Comment:
```suggestion
  if (days >= 7) {
    return amount - secondDiscount;
  }
  
  if (days >= 3) {
    return amount - firstDiscount;
  }
```
You can just immediately return the final value to avoid redundant else...ifs